### DISCUSSION TITLE
salt-cloud man page for section 7

### DISCUSSION THREAD

**Discussion by stappersg** [saltstack/salt#67558] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Getting started with `salt-cloud`

**Describe the solution you'd like**
Manual page of `salt-cloud` says I should see also `salt-cloud(7)`.
In section 7 of the manual pages I would like to find _the guided tour_ of `salt-cloud`.
Examples, working and reproducable examples, of creating a few VMs.
With `libvirt` can be shown how to use `salt-cloud`.

**Describe alternatives you've considered**
_websearch_
Yes, it would be good thing if https://github.com/saltstack/salt/blob/master/doc/man/salt-cloud.7
provides links to _salt-cloud guided tour_.

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://join.slack.com/t/saltstackcommunity/shared_invite/zt-3av8jjyf-oBQ2M0vhXOhJpNpRkPWBvg)
  - [IRC on LiberaChat](https://web.libera.chat/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!

**Comment by waynew** [comment]
@stappersg thanks for the report!

I think I'd consider this a docs bug. One can clone the repo and run:

```
man doc/man/salt-cloud.1
```

At the bottom of the file in the SEE ALSO section it has

```
salt-cloud(7) salt(7) salt-master(1) salt-minion(1)
```

However

```
❯ man doc/man/salt-cloud.7
man: doc/man/salt-cloud.7: No such file or directory
No manual entry for doc/man/salt-cloud.7
```

Though the rest of the files all exist (i.e. salt.7, salt-master.1, salt-minion.1, all in `doc/man/`).

I know that sphinx is somehow involved in building the man pages (we don't do it manually), so there's probably some flag or something that isn't being set.

**Comment by priyank-R** [comment]
@waynew , from what I see, sphinx is buidling the salt-cloud's man page from doc/ref/cli/salt-cloud.rst, which does mention the line  _:manpage:\`salt-cloud(7)\`_ 

But I don't see that page being generated in doc/man directory.
Looking at doc/conf.py, I don't see any mention of salt-cloud(7) in the list/array `man_pages` in that file.

Am I missing something here ? I'd like to help on this one.
